### DISCUSSION TITLE
Show compile commands for file

### DISCUSSION THREAD

**Discussion by MichaelAxtmann** [joaotavora/eglot#1156] [Q&A]
Hello,

I try to find a problem with my includes. Compilation works. However, clangd does not seem to find the include. Is there a way to show the compile commands for the file to check whether the compile_commands.json file has been read in correctly and all -I includes are available?

Best,
Michael

**Comment by joaotavora** [comment]
I sometimes have this problem, too. I switch to the Eglot events buffer, where you may also find `clangd`'s stderr output.  If you look carefully, you will notice where `clangd` informs you what compilation flags it is taking into account for each file.  

Maybe you have to have a verbose invocation of `clangd` for that.  I can't remember.  If that's true, you can do that easily in a one-off way with `C-u M-x eglot`.
